p2panda-rs/src/operation/operation.rs

```diff
@@ -452,6 +453,20 @@ impl From<&OperationEncoded> for Operation {
     }
 }
 
+impl PartialEq for Operation {
+    fn eq(&self, other: &Self) -> bool {
+        self.id == other.id
+    }
+}
+
+impl Eq for Operation {}
+
+impl StdHash for Operation {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.id.hash(state);
```

Comment:
`id` is sometimes `None`, right? This would lead to the same hash for all CREATE operations.

Reply:
Also per specification the hash of an Operation is not only on the `id` but the whole payload.

Reply:
ugh, sorry, this was an id confusion brain fart on my part (I actually already started some work to remove the deprecated `id`... how did I do this then?? :rofl:)